### PR TITLE
PLAT-139238: Add ts/tsx conditions for linting Typescript files

### PR DESCRIPTION
**Pull request by hong6316**
### Issue Resolved / Feature Added
Add ts/tsx conditions for linting typescript code.

### Resolution
Add ts/tsx conditions in commands/lint.js

### Links
PLAT-139238

### Comments
Enact-DCO-1.0-Signed-off-by: Taeyoung Hong (taeyoung.hong@lge.com)

### CODE REVIEW

**Comment by MikyungKim** [comment]
Please add sign off.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## unreleased
+
+### lint
+
+* Added ts/tsx conditions for linting typescript codes.
```
Comment (MikyungKim) [inline CHANGELOG.md:5]:
```suggestion
* Added conditions for linting Typescript files.
```

**Comment by hong6316** [comment]
> Please add sign off.

OK

**Review by MikyungKim — APPROVED** [review]
:lgtm:

**Pull request merged**